src/pages/settings/Wallet/WalletPage/WalletPage.js

```diff
@@ -275,7 +271,7 @@ function WalletPage({bankAccountList, betas, cardList, fundList, isLoadingPaymen
             return;
         }
         PaymentMethods.openWalletPage();
-    }, [network.isOffline]);
+    }, [network.isOffline, bankAccountList, cardList]);
```

Comment:
```suggestion
    }, [network.isOffline, bankAccountList, cardList, fundList]);
```

Reply:
@Pujan92 can you apply this change? I am checklisting right now

The bug still happens in credit card case


https://github.com/Expensify/App/assets/108292595/3304c787-7ac8-4abd-a91a-33709a01f29b

Reply:
Done.